src/datoso/helpers/__init__.py

```diff
@@ -95,7 +95,12 @@ def copy(origin, destination):
         """ Copy file to destination. """
         os.makedirs(os.path.dirname(destination), exist_ok=True)
         try:
-            shutil.copy(origin, destination)
+            if os.path.isdir(origin):
```

Comment:
With this validation, these lines are not needed anymore:
```
        except IsADirectoryError:
            with suppress(FileNotFoundError):
                shutil.rmtree(destination)
            shutil.copytree(origin, destination)
```
https://github.com/laromicas/datoso/pull/3/files#diff-3184a2bafca365ce9b1c340f21ee6add7c2718a1bb6cf4440e857d4f0b02a9a9L101-R109

Reply:
Yep, I left them in while preparing these as a failsafe in case of platform differences causing unexpected behaviours in the original case. If you're happy to pull them out, that's fine.